lib/senders/email_service.js

```diff
@@ -25,7 +25,8 @@ module.exports = (config) => {
         body: {
           text: emailConfig.text,
           html: emailConfig.html
-        }
+        },
+        provider: config.emailService.provider
```

Comment:
(longer-term we will want to set this on *some* requests, but it will need a more intricate control mechanism than a simple pref; for instance, one use case we talked about was the possibility of selectively diverting certain users who are experiencing deliverability problems via Sendgrid or whatever)

Reply:
Alternatively, I'd also be happy with us always setting `provider` like you do here, then making it non-optional and removing the setting from `fxa-email-service`. Either is good, I just don't want there to be two settings is all.

Reply:
> longer-term we will want to set this on some requests, but it will need a more intricate control mechanism than a simple pref

In this case I thought we could just add an optional argument to the `emailService.sendMail` function that would overwrite the config.

Reply:
> selectively diverting certain users who are experiencing deliverability problems

I think this is key 🗝 !